iceoryx_posh/test/moduletests/test_popo_subscriber_port.cpp

```diff
@@ -87,13 +100,22 @@ TEST_F(SubscriberPortSingleProducer_test, initialStateNoChunksLost)
     EXPECT_FALSE(m_sutUserSideSingleProducer.hasLostChunksSinceLastCall());
 }
 
-TEST_F(SubscriberPortSingleProducer_test, initialStateReturnsNoCaProMessage)
+TEST_F(SubscriberPortSingleProducer_test, initialStateReturnsNoCaProMessageWhenNoSubOnCreate)
 {
     auto maybeCaproMessage = m_sutRouDiSideSingleProducer.tryGetCaProMessage();
 
     EXPECT_FALSE(maybeCaproMessage.has_value());
 }
 
+TEST_F(SubscriberPortSingleProducer_test, initialStateReturnsSubCaProMessageWithDefaultOptions)
+{
+    auto maybeCaproMessage = m_sutRouDiSideDefaultOptions.tryGetCaProMessage();
+
+    EXPECT_TRUE(maybeCaproMessage.has_value());
```

Comment:
Please use ``ASSERT_TRUE`` otherwise we could enter undefined behavior land in the next line.